refinery/core/serializers.py

```diff
@@ -34,13 +34,17 @@ def get_is_owner(self, data_set):
                 logger.error("Request is missing a user: %s", e)
                 return False
             return user_request == owner
+        else:
```

Comment:
fixes bug

Reply:
Maybe just `return` in the first `try`? And you could do the same thing in the `except`:
```python
     def get_is_owner(self, data_set):
         try:
             return data_set.is_owner
         except:
             owner = data_set.get_owner()
             try:
                 return owner == self.context.get('request').user
             except AttributeError as e:
                logger.error("Request is missing a user: %s", e)
                return False
```